__init__.py

```diff
@@ -4,8 +4,20 @@
 # https://stefanoflore.it
 # https://ai-wiz.art
 
+# Weight min, max and step values
+WEIGHT_MIN = 0
+WEIGHT_MAX = 1.95
+WEIGHT_STEP = 0.1
+
+# Weight display format (number or slider)
+WEIGHT_DISPLAY = "number"
+
+######## DO NOT MODIFY BELOW THIS LINE ########
+
 import os
 
+WEIGHT_LABEL_SUFFIX = " (weight)"

```

Comment:
This is the suffix added to the "weight" fields inside the node. Originally, this was "_weight" but since it gets included in the actual label displayed inside the UI, I changed this to a more sensible value.